### DISCUSSION TITLE
Keymap context

### DISCUSSION THREAD

**Discussion by trivial-zero** [zed-industries/zed#12326] [Feature Requests]
Where can I find the full list of keymap's `context` field doc? Also I am confused about the whole mapping mechanism. I see you can pass an array, but sometime it is used as a sequence of commands but sometime it is like passing arguments to a command of which again, no doc. Any help appreciated!

**Comment by davidwebca** [comment]
I stumbled into this too. I don't think an array of commands is possible, or at least doesn't work over here. I'd love to swap between two files like most other editors do nowadays (ctrl+tab goes to the previous file, ctrl+tab goes to the previous first file again, and doing so third time will start cycling through the rest of the tabs history). That's so intuitive to me now that I can't live without it since I'm basically swapping between two files most of the day. I've found that hitting cmd+p twice does this, so I've mapped alt+tab to do the same by hitting it twice by adding the same keymap in those two different contexts.

But the best solution would be to enable adding arrays of tasks to run in sequence and hitting twice here would be avoided. 

```jsonc
[
    {
        "context": "Workspace",
        "bindings": {
            "alt-tab": "file_finder::Toggle"
        }
    },
    {
        "context": "FileFinder",
        "bindings": {
            "alt-tab": "file_finder::SelectNext"
        }
    }
]
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
